### DISCUSSION TITLE
any way we can enable remote control while not viewing it?

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#451] [General]
Here is my situation:  I am using skyhdremote.html in the custom remote folder on github for my 500.  The stb is, however, not sky stb, but a Japanese satellite stb.  I have modified the skyhdremote.html so that it send the correct code for my Japanese satellite stb.  However, there are more buttons on the physicial remote than those on the sky hd virtual remote.  Therefore, I need to omit some buttons.  

Before I switch to skyhdremote.html, I used a txt remote which has all the buttons.  It is not as pretty as the skyhdremote.html, though.  I was wondering if I can set up two entries in the config.ini for the same stb, or slingbox, and when I watch video on profile A, I hope to be able to use the remote of profile A, and profile B.  That way I can choose which remote to use, the fancier skyhdremote.html, or the txt remote which has all the buttons.

 Is there any way to achieve this?

**Comment by GerryDazoo** [comment]
Yes there is. Checkout the unified_config.ini example. Just put in two entries pointing to the same slingbox but with different Remote= lines.
